instrumentation/opentelemetry-instrumentation-psycopg/src/opentelemetry/instrumentation/psycopg/__init__.py

```diff
@@ -381,6 +381,46 @@ def callproc(self, *args: Any, **kwargs: Any):
     return TracedCursorFactory
 
 
+class AsyncCursorTracer(CursorTracer):
+    async def traced_execution(
+        self,
+        cursor: dbapi.CursorT,
+        query_method: Callable[..., Any],
```

Comment:
Can you update the annotation here to make it more explicit that the result needs to be awaited?
```suggestion
        query_method: Callable[..., Awaitable[Any]],
```

Reply:
done!